bentoml/_internal/utils/buildx.py

```diff
@@ -35,8 +36,10 @@ def health() -> None:
     """
     cmds = DOCKER_BUILDX_CMD + ["--help"]
     try:
-        output = subprocess.check_output(cmds, stderr=subprocess.STDOUT)
-        assert "--builder string" in output.decode("utf-8")
+        output = subprocess.check_output(cmds, stderr=subprocess.STDOUT).decode(
+            sys.stdout.encoding
+        )
+        assert "--builder string" in output
```

Comment:
@aarnphm just curious, in what case the output may not contain this? and should we provide a better error message here?

Reply:
and we should provide better error message here for sure